services/PredictionService.py

```diff
@@ -1,39 +1,23 @@
-from datetime import datetime
-from sqlalchemy.sql.sqltypes import String
 from controllers import db
-from models.AccountModel import Account
-from models.DataModel import Data
-from models.PredictionModel import Prediction
-from models.QAModel import QAModel
-from models.VisitorModel import Visitor
-from services.DataService import DataService
-from services.QAModelService import QAModelService
-import string
+from models import Prediction
 
 
 class PredictionService():
-    db_conn = db
-    data_service = DataService()
-    model_service = QAModelService()
 
     def create_prediction(self, prediction:Prediction)->Prediction:
-        self.db_conn.session.add(prediction)
-        self.db_conn.session.commit()
-        saved_prediction = Prediction.query.filter(Prediction.uuid == prediction.uuid).first()
+        db.session.add(prediction)
+        db.session.commit()
+        saved_prediction = Prediction.query.filter(Prediction.id == prediction.id).first()
         return saved_prediction
```

Comment:
I suspect this is where we are going to need to call the model_service like Luke mentioned to actually get a prediction.  We should do that before saving a prediction.  Also, we are going to need to use the data_service object to save a new data row here.  Unless SQL Alchemy creates a new Data object in our db automatically because we have defined the relationships in that way?   I don't know enough about SQL Alchemy to know if that's the case or not.  In any event, we want to create a new data row anytime we make a prediction with a novel tweet/question pair.

Reply:
@zacherysmith115 If you want you can merge branch `run_model` into this one so you can integrate. I can help with that tonight if you want